.readthedocs.yml

```diff
@@ -6,3 +6,12 @@ python:
   install:
     - requirements: requirements/pip.txt
     - requirements: requirements/docs.txt
+
+search:
+  ranking:
+    # Deprecated content
+    api/v1.html: -1
+    config-file/v1.html: -1
+
+    # Internal documentation
+    development/design/*: -1
```

Comment:
Probably want this higher at -9 or so :)

Reply:
Lower you mean p: we can test how it goes and see if it's needed.